gemini/sample-apps/finance-advisor-spanner/graph_viz.html

```diff
@@ -0,0 +1,155 @@
 "triangle", "title": "T\u0026D Holdings Inc"}, {"color": "#97c2fc", "id": "8105634904336760832", "label": "Zijin Mining Group Co Ltd", "shape": "triangle", "title": "Zijin Mining Group Co Ltd"}, {"color": "#97c2fc", "id": "8267764490922098688", "label": "Unimicron Technology Corp", "shape": "triangle", "title": "Unimicron Technology Corp"}, {"color": "#97c2fc", "id": "8483937273035882496", "label": "Kasikornbank Public Co Ltd", "shape": "triangle", "title": "Kasikornbank Public Co Ltd"}, {"color": "#97c2fc", "id": "8916282837263450112", "label": "Other Technology", "shape": "triangle", "title": "Other Technology"}, {"color": "#97c2fc", "id": "9132455619377233920", "label": "Ping An Insurance (Group) Co. of China Ltd", "shape": "triangle", "title": "Ping An Insurance (Group) Co. of China Ltd"}, {"color": "red", "id": "576460752303423488", "label": "Consumer Cyclical", "shape": "square", "title": "Consumer Cyclical"}, {"color": "red", "id": "1729382256910270464", "label": "Basic Materials", "shape": "square", "title": "Basic Materials"}, {"color": "red", "id": "2305843009213693952", "label": "Financial Services", "shape": "square", "title": "Financial Services"}, {"color": "red", "id": "3458764513820540928", "label": "Energy", "shape": "square", "title": "Energy"}, {"color": "red", "id": "4035225266123964416", "label": "Technology", "shape": "square", "title": "Technology"}, {"color": "red", "id": "5188146770730811392", "label": " Financial Services", "shape": "square", "title": " Financial Services"}, {"color": "red", "id": "5764607523034234880", "label": "Industrials", "shape": "square", "title": "Industrials"}, {"color": "red", "id": "6917529027641081856", "label": "6917529027641081856", "shape": "square", "title": null}, {"color": "red", "id": "7493989779944505344", "label": "Communication Services", "shape": "square", "title": "Communication Services"}, {"color": "red", "id": "8646911284551352320", "label": "Real Estate", "shape": "square", "title": "Real Estate"}, {"color": "green", "id": "4816177589007679488", "label": "Schroder International Selection Fund Emerging Asia Z Accumulation EUR", "shape": "dot", "title": "Schroder International Selection Fund Emerging Asia Z Accumulation EUR"}, {"color": "green", "id": "5772207347405422592", "label": "Invesco Funds - Invesco Asian Equity Fund E Accumulation EUR", "shape": "dot", "title": "Invesco Funds - Invesco Asian Equity Fund E Accumulation EUR"}, {"color": "green", "id": "7498774854548586496", "label": "Robeco Asian Stars Equities D \u20ac", "shape": "dot", "title": "Robeco Asian Stars Equities D \u20ac"}, {"color": "green", "id": "7842174326135586816", "label": "Manulife Global Fund - Asian Equity Fund AA", "shape": "dot", "title": "Manulife Global Fund - Asian Equity Fund AA"}, {"color": "green", "id": "9108600615101005824", "label": "Allianz Global Investors Fund - Allianz Oriental Income A EUR", "shape": "dot", "title": "Allianz Global Investors Fund - Allianz Oriental Income A EUR"}]);
```

Comment:
Can this JSON be moved to its own file?

Reply:
File is autogenerated by pyviz. so removed it from the codebase